armi/settings/__init__.py

```diff
@@ -151,34 +151,3 @@ def promptForSettingsFile(choice=None):
         choice = int(input("Enter choice: "))
 
     return files[choice]
-
-
-def getMasterCs():
-    """
-    Return the global case-settings object (cs).
-
-    This can be called at any time to create or obtain the primary Cs, a module-level CS
-    intended to be shared by many other objects.
-
-    It can have multiple instances in multiprocessing cases.
-
-    Returns
-    -------
-    cs : Settings
-        The loaded cs object
-    """
-    cs = Settings.instance
```

Comment:
Hmmm, what of this `Settings.instance` object? Should that go away as well?

Reply:
That's a really interesting point.  Let me try that.

Reply:
Done!